catalog/task/update-infra-deployments/0.1/README.md

```diff
@@ -0,0 +1,23 @@
+# update-infra-deployments task
+
+Clones redhat-appstudio/infra-deployments repository, runs a script obtained from the infra-deployment-update-script key in extraData, generates pull-request for redhat-appstudio/infra-deployments repository.
+
+
+## Parameters
```

Comment:
The spacing is pretty inconsistent here. Most descriptions have spaces at the start and end of them but the other values don't

Reply:
I let the editor pretty it for now.